en/docs/services/create-your-first-service.md

```diff
@@ -16,15 +16,15 @@ To create the service, follow the procedure below:
 
 5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+    ![Resource Configuration](../assets/img/services/configure-api-trigger.png){.cInlineImage-half}
     
     Then click **Save API**. Now you have saved the API trigger that starts your service.
     
 6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
 
     1. Click **HTTP**.
     
-        ![Select HTTP Connection](../assets/img/services/select-http-connection.png)
+        ![Select HTTP Connection](../assets/img/services/select-http-connection.png){.cInlineImage-half}
```

Comment:
This image appears quite large. can we make it as small as possible, without causing any visual inconvenience?

Reply:
Image size is not visible in github view mode. This shows the original image size.